Dfe.Data.SearchPrototype/SearchForEstablishments/SearchResponseStatus.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Dfe.Data.SearchPrototype.SearchForEstablishments;
+
+public enum SearchResponseStatus
+{
+    Success,
+    InvalidRequest,
+    SearchServiceError,
+    DataFormatError
```

Comment:
`DataFormatError` doesn't currently appear to be used.

Out of curiosity, what is the intention for this case? Data is returned from the search API, but the mapper fails to transform/map it out of the Adapter, perhaps? (if so, maybe we need to add try/catch or some other logic to return this?)

Presumably this repo/context doesn't care about the transport encoding (JSON/XML/binary/something else) and everything this side of the adapter is dealing with de-serialised C# objects -- assuming this is correct, I can't quite get my head around what `Format` is considered erroneous at this point.

Reply:
Yes, I added it thinking I was going to use it if the mapping went wrong and then didn't